libs/modulify-ui/src/lib/utils/documentWithJss.tsx

```diff
@@ -0,0 +1,47 @@
+import Document, { DocumentContext, DocumentInitialProps } from 'next/document';
+import { ComponentType } from 'react';
+import { JssProvider, SheetsRegistry, createGenerateId } from 'react-jss';
+
+const SERVER_SIDE_JSS_STYLES_ID = 'server-side-jss-styles';
+
+type DocumentWithInitialProps = ComponentType<any> & {
```

Comment:
I think it will be good either to specify different type if it possible or to disable lint rule for this line.
This will fix style_checks error.